model_xpubs.go

```diff
@@ -80,6 +80,25 @@ func getXpubByID(ctx context.Context, xPubID string, opts ...ModelOps) (*Xpub, e
 	return xPub, nil
 }
 
+// getXPubs will get all the xpubs matching the conditions
+func getXPubs(ctx context.Context, usingMetadata *Metadata, conditions *map[string]interface{},
+	queryParams *datastore.QueryParams, opts ...ModelOps) ([]*Xpub, error) {
+
+	modelItems := make([]*Xpub, 0)
+	if err := getModelsByConditions(ctx, ModelXPub, &modelItems, usingMetadata, conditions, queryParams, opts); err != nil {
+		return nil, err
+	}
+
+	return modelItems, nil
+}
+
+// getXPubsCount will get a count of the xpubs matching the conditions
+func getXPubsCount(ctx context.Context, usingMetadata *Metadata,
+	conditions *map[string]interface{}, opts ...ModelOps) (int64, error) {
+
+	return getModelCountByConditions(ctx, ModelXPub, Xpub{}, usingMetadata, conditions, opts)
+}
```

Comment:
Does opts need the `...`

`opts` vs `opts...`

Not sure if that is an error or not...